torchmdnet/scripts/train.py

```diff
@@ -98,6 +98,8 @@ def get_args():
     parser.add_argument('--wandb-use', default=False, type=bool, help='Defines if wandb is used or not')
     parser.add_argument('--wandb-name', default='training', type=str, help='Give a name to your wandb run')
     parser.add_argument('--wandb-project', default='training_', type=str, help='Define what wandb Project to log to')
+    parser.add_argument('--wandb-resume', default=False, type=bool, help='Defines if wandb run should be resumed')
+    parser.add_argument('--wandb-id', default=None, type=str, help='Define what wandb run to resume, you can find the id from the wandb table')
```

Comment:
You could have just one argument: --wandb-resume-from-id. I see no possibility for wandb-resume to be present when wandb-id is not.